pkgs/development/libraries/miniaudio/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+}:
+stdenv.mkDerivation rec {
+  pname = "miniaudio";
+  version = "unstable-2020-04-20";
+
+  src = fetchFromGitHub {
+    owner = "mackron";
+    repo = "miniaudio";
+    rev = "4d813cfe23c28db165cce6785419fee9d2399766";
+    sha256 = "sha256-efZLZTmkLtvcysd25olDE/QqunU5YTYwSVmUZXPKGIY=";
+  };
+
+  installPhase = ''
+    mkdir -p $out
+    touch $out/SRC_ONLY_DERIVATION
```

Comment:
What is this?

Reply:
That was just to make nix erroring due to the derivation producing no output.

Hmm the pyminiaudio package *does* have a copy of miniaudio.h that we could borrow... but I don't know how reliable that is.

Reply:
Lets not do that. I just didn't double check if it was the right package.

Reply:
Changed the miniaudio derivation to simply copy src to out now.